server/handlers/content_modifier.go

```diff
@@ -30,7 +30,7 @@ func NewContentModifier(token string,
 	}
 }
 
-// TODO: Similar mechanisms for filters and applications
+// AddMetadataForPatterns TODO: Similar mechanisms for filters and applications
 // Takes in response bytes, and add metadata to it based on some checks
```

Comment:
It should be `AddMetadataForPatterns takes in response bytes, and add metadata to it based on some checks`

Reply:
Okay 👍🏼